src/wizard/github.jl

```diff
@@ -8,14 +8,27 @@ function github_auth(;allow_anonymous::Bool=true)
     if !isassigned(_github_auth) || !allow_anonymous && isa(_github_auth[], GitHub.AnonymousAuth)
         # If the user is feeding us a GITHUB_TOKEN token, use it!
         if length(get(ENV, "GITHUB_TOKEN", "")) == 40
-            _github_auth[] = GitHub.authenticate(ENV["GITHUB_TOKEN"])
+            try 
+                _github_auth[] = GitHub.authenticate(ENV["GITHUB_TOKEN"])
+            catch e
+                    println(stdout, "The GitHub token found in \"GITHUB_TOKEN\" is incorrect or has expired.")
+                    if yn_prompt("Would you like to attempt to obtain a new token from GitHub?") == :y
+                        _github_auth[] = GitHub.authenticate(obtain_token())
+                    else
+                        error("No GitHub token available. Please provide one in the \"GITHUB_TOKEN\" environment variable, or reply yes to the previous prompt.")
+                    end
+            end
         else
             if allow_anonymous
                 _github_auth[] = GitHub.AnonymousAuth()
             else
                 # If we're not allowed to return an anonymous authorization,
                 # then let's create a token.
-                _github_auth[] = GitHub.authenticate(obtain_token())
+                if yn_prompt("Would you like to attempt to obtain a new token from GitHub?") == :y
```

Comment:
As `obtain_token()` already requires user interaction to proceed I don't really see the point of adding an additional prompt.

Reply:
@staticfloat are you ok with no additional prompt? I can revert back to no extra prompt if so. This would be the original behavior with the GitHub prompt but no extra "would you like to proceed" step.

Reply:
Yeah, @omus knows this part better than I do; let's do what he suggests.